pkg/crc/preflight/preflight.go

```diff
@@ -103,7 +104,7 @@ func doPreflightChecks(config crcConfig.Storage, checks []Check) error {
 
 func doFixPreflightChecks(config crcConfig.Storage, checks []Check, checkOnly bool) error {
 	for _, check := range checks {
-		if check.flags&CleanUpOnly == CleanUpOnly {
+		if check.flags&CleanUpOnly == CleanUpOnly || check.flags&StartUpOnly == StartUpOnly {
```

Comment:
Note: This reverts 60c286c9, and the code before the revert looks the same, so this change looks good :)